tools/osbuilder/node-builder/azure-linux/package_install.sh

```diff
@@ -62,6 +63,8 @@ cp -a --backup=numbered src/runtime/data/kata-collect-data.sh "${PREFIX}/${DEBUG
 echo "Installing shim binary and configuration"
 cp -a --backup=numbered src/runtime/containerd-shim-kata-v2 "${PREFIX}/${SHIM_BINARIES_PATH}"/"${SHIM_BINARY_NAME}"
 
-cp -a --backup=numbered src/runtime/config/"${SHIM_CONFIG_FILE_NAME}" "${PREFIX}/${SHIM_CONFIG_PATH}/${SHIM_CONFIG_INST_FILE_NAME}"
+if [ "${SHIM_REDEPLOY_CONFIG}" == "yes" ]; then
```

Comment:
Fully in line with this PR but just one note: look at "Installing SNP shim debug configuration" - there is another instance where a config gets overwritten :)

Reply:
Line 64? That would be the shim binary that is overwritten, which is expected?

Reply:
Referring to these lines above:
```
echo "Installing SNP shim debug configuration"
	cp -a --backup=numbered src/runtime/config/"${SHIM_DBG_CONFIG_FILE_NAME}" "${PREFIX}/${SHIM_CONFIG_PATH}"/"${SHIM_DBG_CONFIG_INST_FILE_NAME}"
```

Reply:
Oh! Will fix that.

Reply:
Done